src/components/PreviewOverlay.tsx

```diff
@@ -63,6 +63,7 @@ function PreviewOverlay(): JSX.Element {
           onClick={() => {
             toggleOverlay();
             defaultCameraView();
+            pauseApparatus();
```

Comment:
Function pauseApparatus() should also be used in Overlay.tsx for when the user presses the ESC key while previewing.